contracts/tokenfactory/src/contract.rs

```diff
@@ -59,6 +64,7 @@ pub fn query(deps: Deps<NeutronQuery>, _env: Env, msg: QueryMsg) -> NeutronResul
             subdenom,
         } => to_binary(&query_full_denom(deps, creator_addr, subdenom)?)?,
         QueryMsg::DenomAdmin { subdenom } => to_binary(&query_denom_admin(deps, subdenom)?)?,
+        QueryMsg::BeforeSendHook { denom } => to_binary(&query_before_send_hook(deps, denom)?)?,
```

Comment:
what about sudo `BlockBeforeSend` and `TrackBeforeSend` messages?

Reply:
they both are the part of inside logic, not  "api". also not the part of task implemented in this pr